nemo/utils/exp_manager.py

```diff
@@ -872,6 +879,20 @@ def configure_loggers(
         logger_list.append(clearml_logger)
         logging.info("ClearMLLogger has been set up")
 
+    if create_neptune_logger:
+        if neptune_kwargs is None:
+            neptune_kwargs = {}
+        if "name" not in neptune_kwargs and "project" not in neptune_kwargs:
+            raise ValueError("name and project are required for neptune_logger")
+        if "api_key" not in neptune_kwargs and not os.getenv("NEPTUNE_API_TOKEN", None):
+            raise ValueError(
+                "either api_key should be set in neptune_kwargs or NEPTUNE_API_TOKEN should be set in environment variable for neptune_logger"
+            )
```

Comment:
Havent checked for import error here, as PyTorchLightning by default checks for the same within its NeptuneLogger wrapper.